src/components/index.js

```diff
@@ -1,5 +1,8 @@
 export * from 'theme-ui';
 
+export * from './position';
+export BackgroundImage from './backgroundImage';
+export Input from './input';
 export IconButton from './iconButton';
 export Truncate from './truncate';
-export ReadMore from './readMore';
\ No newline at end of file
+export ReadMore from './readMore';
```

Comment:
Don't delete the newline at the end of the file. 
Newline at EOF is recommended. Check out [this link](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) to know more